<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgets_RIADClass</class>
 <widget class="QMainWindow" name="QtWidgets_RIADClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIAD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(213, 213, 213);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_ori">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>500</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>500</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(115, 115, 115);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_rec">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>500</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>500</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(115, 115, 115);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 58, 58);</string>
        </property>
        <widget class="QLabel" name="lbl_heatmap">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>296</width>
           <height>296</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(115, 115, 115);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_score">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>340</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>得分：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_result_str">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>380</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>判定结果：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_result">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>380</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_result_str_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>420</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>执行时间(ms)：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_time">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>420</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>450</y>
           <width>281</width>
           <height>281</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(213, 213, 213);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>执行</string>
            </attribute>
            <widget class="QPushButton" name="btn_inference">
             <property name="geometry">
              <rect>
               <x>580</x>
               <y>20</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>执行推理</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_openfile">
             <property name="geometry">
              <rect>
               <x>520</x>
               <y>20</y>
               <width>51</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>491</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>调参</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1326</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgets_RIAD.qrc"/>
 </resources>
 <connections/>
</ui>
